### DISCUSSION TITLE
Light sensor compatiblity

### DISCUSSION THREAD

**Discussion by MonsieurThom** [AutoDarkMode/Windows-Auto-Night-Mode#524] [Q&A]
Hello, does this app can work with a USB light sensor? And it would adapt the themes to the ambient lighting.

**Comment by Spiritreader** [comment]
The app itself has no light sensor integration, no.
But if your light sensor has an API you can use that to steer AutoDarkMode either with custom scripts (if your sensor has command line support) or creating a module/event (if it has a C# api, which would add support for that sensor)

**Comment by Jay-o-Way** [comment]
@MonsieurThom latest version has light sensor response built in. Can you check if the usb sensor works?
